Userland/Libraries/LibC/netdb.cpp

```diff
@@ -10,6 +10,8 @@
 #include <AK/ScopeGuard.h>
 #include <Kernel/Net/IPv4.h>
 #include <arpa/inet.h>
+#include <cstdio>
+#include <cstring>
```

Comment:
we've already got `<stdio.h>` and `<string.h>` included below, and we only include STL headers when absolutely required (like `<initalizer_list>` and `<new>`). So these should be removed.